microsoft-edge/webview2/samples/wv2deploymentvsinstallersample.md

```diff
@@ -95,6 +77,65 @@ Microsoft Visual Studio is required.  Microsoft Visual Studio Code is not suppor
 <!-- 1. Close the sample app window. -->
 
 
+<!-- ====================================================================== -->
+## Step 5 - Create a Visual Studio installer for the Evergreen WebView2 Runtime
+
+This section highlights deployment workflows included in [Distribute your app and the WebView2 Runtime](/microsoft-edge/webview2/concepts/distribution#deploying-the-evergreen-webview2-runtime) _Deploying the Evergreen WebView2 Runtime_ to complete the following tasks:
+
+* Download the Evergreen WebView2 Runtime Bootstrapper using a link.
+* Package the Evergreen WebView2 Runtime Bootstrapper.
+* Package the Evergreen WebView2 Runtime Standalone Installer.
+
+### Build Steps
+
+Complete these steps to edit and run the project.
+
+1. Start Visual Studio.
+
+1. Open the local copy of the `WebView2Samples` repo.
+
+1. Edit the `product.xml` file depending on the workflow you wish to use.
+
+    * For "Package the Evergreen WebView2 Runtime Bootstrapper",
+        * Within the `<PackageFiles CopyAllPackageFiles="false">` and `</PackageFiles>` section, un-comment the line `<PackageFile Name="MicrosoftEdgeWebview2Setup.exe" />` and comment out other lines.
+        * Within the `<Commands Reboot="Defer">` and `</Commands>` section, make sure `PackageFile` points to `"MicrosoftEdgeWebview2Setup.exe"` so that the VS installer is using the Bootstrapper.
+
+    * For "Download the Evergreen WebView2 Runtime Bootstrapper through link",
+        * Within the `<PackageFiles CopyAllPackageFiles="false">` and `</PackageFiles>` section, un-comment the line `<PackageFile Name="MicrosoftEdgeWebview2Setup.exe" HomeSite="WebViewRuntime" PublicKey="..."/>` and comment out other lines. Note that the `PublicKey` for the WebView2 Runtime Bootstrapper may change without notice and we are working on addressing this issue. For now, you may need to replace it with an updated `PublicKey`.
+        * Within the `<Commands Reboot="Defer">` and `</Commands>` section, make sure `PackageFile` points to `"MicrosoftEdgeWebview2Setup.exe"` so that the VS installer is using the Bootstrapper.
```

Comment:
```suggestion
        * Within the `<Commands Reboot="Defer">` and `</Commands>` section, make sure `PackageFile` points to `"MicrosoftEdgeWebview2Setup.exe"` so that the Visual Studio installer is using the Bootstrapper.
```